packages/cardano-services/src/Program/loadHttpServer.ts

```diff
@@ -34,6 +34,7 @@ import pg from 'pg';
 export interface HttpServerOptions extends CommonProgramOptions {
   serviceNames?: ServiceNames[];
   enableMetrics?: boolean;
+  buildInfo?: string;
```

Comment:
```suggestion
  meta?: ServerMetadata;
```

Considering we're just mapping this value to the endpoint, terminology alignment makes sense. Also, it's not strictly build information, given there's `startupTime`.

I suggest renaming all uses of `buildInfo` to `meta`

Reply:
> Then we [pass it down](https://github.com/input-output-hk/cardano-js-sdk/pull/573/files#diff-cfc78f77666af1469e2e670c258fecabed5a923995563490c52370f7a6b4888cR211) to the config.buildInfo and then [compose](https://github.com/input-output-hk/cardano-js-sdk/pull/573/files#diff-db63771378e2c7af74fab64645861ab1f0556c3320ce46fabf60eedf796ab08eR133-R134) the matadata : ServerMetadata obj as the final endpoint response (buildInfo + startupTime).

Ah yes, fine as is then :+1: